python/hail/context.py

```diff
@@ -636,6 +636,46 @@ def read(self, path, drop_samples=False, drop_variants=False):
             self,
             self._jhc.read(path, drop_samples, drop_variants))
 
+    @handle_py4j
+    @typecheck_method(path=oneof(strlike, listof(strlike)))
+    def get_vcf_metadata(self, path):
```

Comment:
I'm not sure how @tpoterba but I think this is great.  Good work.  Minor suggestion, feel free to ignore if you disagree: I prefer load to get, since get sounds more like an accessor.

Reply:
I think this looks awesome. It's simple and does exactly what it needs to do.